test/unit/index.test.ts

```diff
@@ -5,14 +5,7 @@ import { expect } from 'chai';
 import * as mongodb from '../../src/index';
 import { setDifference } from '../mongodb';
 
-/**
```

Comment:
what's the motivation for the changes in this file?

Reply:
https://github.com/mongodb/node-mongodb-native/pull/4292#pullrequestreview-2408002618

I kinda covered it in here, let me know if I can clarify something.

Reply:
ts-node internally compiles the typescript before running the javascript, because we didn't explictily mark 'AnyBulkWriteOperation', 'BulkWriteOptions' as `type` so ts-node leaves behind empty exports with those names, our real JS in lib never had those, so this was a bit of cruft only related to how our tests run.